rules/Php81/Rector/Array_/FirstClassCallableRector.php

```diff
@@ -89,59 +82,14 @@ public function name()
      */
     public function getNodeTypes(): array
     {
-        return [Property::class, ClassConst::class, Array_::class, FuncCall::class];
+        return [Property::class, ClassConst::class, Array_::class];
     }
 
     /**
-     * @param Property|ClassConst|Array_|FuncCall $node
+     * @param Property|ClassConst|Array_ $node
```

Comment:
I wonder about the remaining functionality.. instead of separating the FuncCall shouldn't the whole rule be moved to CodingStyle instead?

Reply:
that will be bc break, most methodcall/static call movement seems fine, just for func call, that seems a bit too much

Reply:
I see. It feels a bit inconsistent but I am fine with it